pkg/channel/distributed/dispatcher/dispatcher/dispatcher.go

```diff
@@ -329,6 +333,11 @@ func (d *DispatcherImpl) ConfigChanged(ctx context.Context, configMap *corev1.Co
 
 	// Create A New Dispatcher With The New Configuration (Reusing All Other Existing Config)
 	d.Logger.Info("Consumer Changes Detected In New Configuration - Recreating Dispatcher")
+	return d.reconfigure(newConfig)
+}
+
+// Shut down the current dispatcher and recreate it with new settings
+func (d *DispatcherImpl) reconfigure(newConfig *sarama.Config) Dispatcher {
 	d.Shutdown()
```

Comment:
Super nit picky but... can you move this function down below SecretChanged() - keeps the two public "changed" functions together and improves (imo) readability - thanks!

Reply:
absolutely